mkosi/__init__.py

```diff
@@ -3469,6 +3469,13 @@ def find_files(root: Path) -> Iterator[Path]:
     yield from scandir_recursive(root,
                                  lambda entry: Path(entry.path).relative_to(root))
 
+def cpio_binary() -> str:
+    # Some distros (Mandriva) install BSD cpio as "cpio", hence prefer
+    # GNU "cpio" if it exists, which should be GNU cpio wherever it exists.
+    # We are interested in exposing same behaviour everywhere hence
+    # it's preferable to use the same implementation of cpio
+    # everywhere.
```

Comment:
That's not what the code does, though, since it prefers `bsdcpio` if it exists (and on my Arch machines it exists).

Reply:
Could you maybe solve this in Mandriva? Because as the code currently is this would prefer BSD cpio over GNU cpio, but we want to always use GNU cpio when it is available.

Reply:
under which name does the gnu cpio RPM install the binary then under mandriva? it must pick something

Reply:
The comment still doesn't match the code in the latest version…